image/bmp.ksy

```diff
@@ -419,10 +419,7 @@ types:
           cases:
             true: strz
         if: has_profile
-        doc-ref: https://docs.microsoft.com/en-us/previous-versions/windows/desktop/wcs/using-structures-in-wcs-1-0 "If the profile is embedded,
-          profile data is the actual profile, and if it is linked, the profile data is the
-          null-terminated file name of the profile. This cannot be a Unicode string. It must be composed exclusively
-          of characters from the Windows character set (code page 1252)."
+        doc-ref: https://docs.microsoft.com/en-us/previous-versions/windows/desktop/wcs/using-structures-in-wcs-1-0 "If the profile is embedded, profile data is the actual profile, and if it is linked, the profile data is the null-terminated file name of the profile. This cannot be a Unicode string. It must be composed exclusively of characters from the Windows character set (code page 1252)."
```

Comment:
```suggestion
        doc-ref: >-
          https://docs.microsoft.com/en-us/previous-versions/windows/desktop/wcs/using-structures-in-wcs-1-0
          "If the profile is embedded, profile data is the actual profile, and if it is linked,
          the profile data is the null-terminated file name of the profile.
          This cannot be a Unicode string. It must be composed exclusively of characters
          from the Windows character set (code page 1252)."
```

Reply:
I feel like it may be better to put the text into `doc` and leave `doc-ref` for the link only

Reply:
> I feel like it may be better to put the text into `doc` and leave `doc-ref` for the link only

Yes, you're right. I intended to emphasize the fact that it is a quote from that specific URL, but it really should be rather in `doc`.

Also, I've just found that the link address no longer works (Microsoft does a bad job of keeping URLs permanent) and it has moved to https://docs.microsoft.com/en-us/windows/win32/wcs/using-structures-in-wcs-1-0.